server/server.js

```diff
@@ -245,6 +300,7 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
       (this.subs[to] ? 'is subscribed' : 'not subscribed')
     );
 
+    this._runPostFilters(socket, message, messageType);
```

Comment:
I don't get the point of post filter. The only things it is 'post' is nameSync and getResource? What's the point of that?